### ISSUE TITLE
Use web workers for storing data

### ISSUE THREAD

**Issue by icidasset** [icidasset/isotach#14]
Otherwise the browser hangs 🤦‍♂️ 

- [x] Local
- [x] RemoteStorage
- [ ] Blockstack

Related issues:
- https://github.com/blockstack/blockstack-storage-js/issues/12

**Comment by icidasset** [comment]
On hold, waiting on a Blockstack update.
Anonymous doesn't suffer from this issue anyway.

**Comment by icidasset** [comment]
A web worker has been set up for the local auth type, I still need to extract the generic parts and apply the same stuff for Blockstack. Blockstack stuff is still on hold though because it can't be used inside a web worker yet.
